app/models/miq_action.rb

```diff
@@ -742,7 +742,7 @@ def action_container_image_annotate_deny_execution(action, rec, inputs)
       return
     end
 
-    unless rec.digest.present?
+    unless rec.kind_of?(OpenshiftContainerImage)
```

Comment:
Since the original code is OpenShift specific already, maybe we can let this go, but it would be preferable if we could avoid provider-specifics.  @simon3z Thoughts?